app/src/main/java/org/commonvoice/saverio/MainActivity.kt

```diff
@@ -767,6 +768,27 @@ class MainActivity : VariableLanguageActivity(R.layout.activity_main) {
         )
     }
 
+    fun setSaveLogFile(status: Boolean) {
```

Comment:
I'm pretty sure that we can avoid adding this code since you created the LogPrefManager.
I know that other things inside SettingsFragment use a reference to MainActivity to access and change preferences, but this is being fixed and you should just use the "correct" way.